### PR TITLE
CEDS-2327

### PR DESCRIPTION
**Pull request by alsahid1993**
https://jira.tools.tax.service.gov.uk/browse/CEDS-2327
Made Page GDS compliant

### CODE REVIEW

File: app/views/declaration/exporter_details.scala.html [hunk 1]
```diff
@@ -21,49 +21,150 @@
...
+        form("details.address.country"),
+        label = messages("supplementary.address.country"),
+        None,
+        None,
+        messages("declaration.originationCountry.empty"),
+        AutoCompleteItem.fromCountry(Countries.allCountries, _.countryCode)
+    )
+}
+
+@fullName = {
```
Comment (PauloPortugal) [inline app/views/declaration/exporter_details.scala.html:90]: I thought we were supposed to use @iservport address details approach. 
Please check it out.

**Comment by PauloPortugal** [comment]
You also need to run 
```
sbt scalafmt test:scalafmt
```

File: app/views/declaration/exporter_details.scala.html (hunk 1 above)
Comment (alsahid1993) [reply app/views/declaration/exporter_details.scala.html]: @iservport's approach has been utilised

File: conf/messages.en [hunk 2]
```diff
@@ -317,6 +317,7 @@ supplementary.consignor.eori = Exporter’s EORI number
 supplementary.consignor.address.info = Exporter’s name and business address
 supplementary.exporter-details.help-item1 = Exporter’s EORI number, go to data element 3/2 Exporter Identification Number
 supplementary.exporter-details.help-item2 = Exporter’s name and business address, go to data element 3/1 Exporter
+supplementary.exporter-details-name-address = Exporter’s name and address
```
Comment (PatrykRudnicki) [inline conf/messages.en:320]: For all new messages it's better to use `declaration.exporter-details` etc. Supplementary is old thing when we had just one journey. During Changing the page we're trying to get rid of `supplementary` and replace it with `declaration`.

File: test/views/declaration/ExporterDetailsViewSpec.scala [hunk 3]
```diff
@@ -136,7 +136,7 @@ class ExporterDetailsViewSpec extends UnitViewSpec with CommonMessages with Stub
 
         view must haveGovukGlobalErrorSummary
         view must containErrorElementWithTagAndHref("a", "#details")
-        view.getElementsByClass("govuk-list govuk-error-summary__list").text() must include(messages("supplementary.namedEntityDetails.error"))
+        view.getElementsByClass("govuk-list govuk-error-summary__list").attr("value") mustBe empty
```
Comment (iservport) [inline test/views/declaration/ExporterDetailsViewSpec.scala:139]: Did you consider the approach `view must containErrorElementWithMessage(...)` ?

Comment (alsahid1993) [reply test/views/declaration/ExporterDetailsViewSpec.scala:137]: I will consider using this approach going forward

**Pull request merged**